examples/storage/cassandra/README.md

```diff
@@ -165,18 +165,18 @@ cassandra   None         <none>        9042/TCP   45s
 
 If an error is returned the service create failed.
 
-## Step 2: Use a Pet Set to create Cassandra Ring
+## Step 2: Use a StatefulSet to create Cassandra Ring
```

Comment:
I vote for `Stateful Set`. I saw similar use in the docs of `Replica Set`, `Scheduled Job`, etc